### PR TITLE
Add a Lightning Terminal guide to the main guide

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

This is a proposal to add Lightning Terminal (with Loop, Pool and Faraday integrated) as part of the main guide! See rationales below for why main guide rather than as a bonus guide, but this is opened for debate of course! :)

### Why

* Lightning Terminal can be run on top of LND as one single daemon that integrates the Loop, Pool and Faraday daemons.

  * **Loop** allows to loop sats in/out of channels (= submarine swaps) via Lightning Labs Loop node.
    * This tool can be very useful to end-users and merchants to quickly and easily manage their channel balances (merchants need to regualrly move sats to remote and onchain, while end-users might need to move back sats to the local side after spending an entire channel). The tool can also be used by routing node operators.

  * **Pool** is a channel marketplace. 
    * This tool can be very useful for merchants to get inbound liqudiity and for routers to earn an income.

  * **Faraday** is an accounting tool for advanced users (CLI-only, requires manipulation of JSON/CSV etc)

* It also provided a web UI than can be accessed inside a user LAN and offers a GUI for the Loop and Pool services.

* Running Lightning Terminal also allows to connect to the web Terminal via the recently developped Lighning Node Connect (LNC). This allows an operator to securely manage its node from anywhere.

* Running Lightning Terminal allows a user to loop in/out directely from the Ride The Lightning web UI

* The RaspiBolt is based on LND (for now) and Lightning Terminal is developped by the same people than LND. We should tehrefore expect continuous development of Lightning Terminal alongside LND and possibly added services in the future. Moreover, running LiT with LND remote avoids the bugs associated with running LiT with LND integrated (as was proposed before as an option in RaspiBolt v2).

TL;DR: Lightning Terminal gives access to very useful lightning services (i.e. Loop and Pool) that helps merchants, end-users and routers to manage their liquidity and also offers helpful dashboard that can be used locally or remotely. LiT should be maintained and improved upon in the future as it is developped by Lightning Labs, one of the main LN company.

#### How

* Added a Lightning Terminal guide in the main section, between LND and Ride the Lightning (and updated the Next links accordingly).
* Added several illustrations in the Images directory
* Updated the Lightning page to add a reference to Lightning Terminal
* Updated the Ride the Lightning guide to add a section on how to configure it to connect to the LiT/Loop server

#### Scope

- [x] significant change to core configuration
- [ ] independent bonus guide
- [ ] simple bug fix

#### Test & maintenance

Test this PR by following the guide and installing Lightning Terminal, visiting the web UIs and trying the first steps of the Pool and Loop examples.

#### Animated GIF (optional)

pool

### CODE REVIEW

**Comment by Stadicus** [comment]
I like the addition to LiT with remote LND a lot. Using the version with integrated LND caused a lot of issues in the old RaspiBolt guide, as it's an alternative to the default installation and not just an addon.

I'm skeptical if making this part of the main guide is a step in the right direction, however. Why?

* I think the RaspiBolt should describe a reference implementation of a modular Bitcoin/Lightning node.
* It should be possible to easily switch out LND for c-lightning in the future.
* The services provided by LiT are of a commercial nature and centralized (IIUC), with many independent alternatives available. I don't mind adding them to the bonus guide if they're useful (nothing bad with making money), but these shouldn't be prominently made part of the core guide.

**Comment by RooSoft** [comment]
Totally agree with @Stadicus here.

**Comment by VajraOfIndra** [comment]
> I'm skeptical if making this part of the main guide is a step in the right direction, however. Why?

You make very valid points. I thought making these tools availble to more people and optimizing RTL was pretty cool but the fact that they are commercial services is definitely an issue.

I transformed the guide as a bonus guide instead. PTAL!

**Comment by VajraOfIndra** [comment]
Closing this PR because of merging conflicts. Re-opening with a new branch.
